ui/lib/pki/addon/templates/certificates.hbs

```diff
@@ -7,4 +7,5 @@
     model=this.model.id
   }}
   @isEngine={{true}}
-/>
\ No newline at end of file
+/>
+{{outlet}}
```

Comment:
It feels slightly redundant to have a `/certificates` template **and** a `certificates/index` template. 
Is there a reason behind making the template below a child route? If the index view will remain the same wondering if we should combine the two files into one - and maybe down the road we can even extract `SecretListHeader` from each index.hbs (if all the list views have the same header) and move it to a parent route

Reply:
Yeah, this was a conversation Jordan and I had when discussing routing. The big reason for this is because the main route files for some of the routes will have different headers. Some won't have SecretListHeader at all they may have, like in the Configuration route a totally different landing page, and having the index file gives us some wiggle room on playing with the {{outlets}}. It's kind of an experiment at first, and if it turns out we don't need to make these two separate we'll move them back together. In planning, this made sense, we'll see how it works in practice.

Reply:
Sounds good! Thanks for the context